composer.json

```diff
@@ -11,7 +11,9 @@
     ],
     "require": {
         "php": ">=5.3",
```

Comment:
alternatively, we should bump version to >=5.5 for now and fix Travis matrix to work only with supported PHP versions